lib/server.js

```diff
@@ -123,6 +142,16 @@ function create(log, error, config, routes, db) {
       }
     )
     server.auth.strategy(
+      // This strategy fetches the sessionToken with its
+      // verification state. It doesn't check that state.
+      'sessionTokenVerified',
```

Comment:
> My understanding is that adding new auth strategies will require updates to clients to
> create Hawk credentials using sessionTokenVerified instead of sessionToken.

I don't think this is true - we recently added a `sessionWithDevice` auth strategy without requiring any client changes.